infra/terraform/test-org/org-iam-policy/iam.tf

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+locals {
+  cft_ci_group          = "cft-ci-robots@test.infra.cft.tips"
+  cft_dev_group         = "cft-developers@dev.infra.cft.tips"
+  gcp_admins_group_test = "gcp-admins@test.infra.cft.tips"
+  gcp_admins_group_dev  = "gcp-admins@dev.infra.cft.tips"
```

Comment:
We shouldn't be granting the dev group any access to the test org. That might have just been a temporary hotfix I never rolled back.

Reply:
removed